packages/react-native-win32/src/Libraries/Components/View/View.win32.js

```diff
@@ -30,15 +30,12 @@ const View: React.AbstractComponent<
   ViewProps,
   React.ElementRef<typeof ViewNativeComponent>,
 > = React.forwardRef((props: ViewProps, forwardedRef) => {
-  // [Win32 Intercept props to warn about them going away
-
-  // $FlowFixMe react-native-win32 doesn't have forked types in Flow yet
-  if (props.acceptsKeyboardFocus !== undefined) {
-    warnOnce(
-      'deprecated-acceptsKeyboardFocus',
-      '"acceptsKeyboardFocus" has been deprecated in favor of "focusable" and will be removed when Office moves to React Native 0.64',
-    );
-  }
+  // [Win32
+  invariant(
+    // $FlowFixMe Wanting to catch untyped usages
+    props.acceptsKeyboardFocus === undefined,
```

Comment:
IIRC the flowconfig we have might actually prevent casting to any in strict files (not certain though)? I remember running into something like that before but don't remember the details.  Generally agree though that it's better to model the type then to suppress the type system.

Since the suppression comment doesn't cover anything beyond the access, I think I'm just going to leave this as is though, since it should be temporary.

Reply:
Fair enough and thanks for the explanation re the flow config 👍